app/code/Magento/Directory/Block/Data.php

```diff
@@ -152,31 +152,38 @@ public function getRegionCollection()
     }
 
     /**
+     * @param null|string $defValue
+     * @param string $name
+     * @param string $id
+     * @param string $title
      * @return string
      */
-    public function getRegionHtmlSelect()
+    public function getRegionHtmlSelect($value = null, $name = 'region', $htmlId = 'state', $title = 'State/Province')
```

Comment:
We are not allowed to add new parameters to public methods according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/). Please, consider creating the new method and deprecate the old method.